capstone/src/main/java/com/google/univiz/api/VisResource.java

```diff
@@ -1,7 +1,20 @@
 package com.google.univiz.api;
```

Comment:
Should I cease work on the VisResource Interface?

Reply:
I believe you no longer need to work on `VisResource`, so yes.